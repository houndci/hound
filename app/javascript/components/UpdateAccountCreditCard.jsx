import React from 'react';

import App from './UpdateAccountCreditCard/components/App';

export default class UpdateAccountCreditCard extends React.Component {
  render() {
    return(
      <App {...this.props} />
    );
  }
}

