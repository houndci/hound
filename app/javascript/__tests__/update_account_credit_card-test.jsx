import UpdateAccountCreditCard from '../components/UpdateAccountCreditCard';

it('renders appropriately', () => {
  const wrapper = shallow(
    <UpdateAccountCreditCard
      authenticity_token={"csrf_token"}
      stripe_customer_id_present={true}
    />
  );
  expect(wrapper).toMatchSnapshot();
});

it('renders appropriately without Stripe customer ID', () => {
  const wrapper = shallow(
    <UpdateAccountCreditCard
      authenticity_token={"csrf_token"}
      stripe_customer_id_present={false}
    />
  );
  expect(wrapper).toMatchSnapshot();
});

