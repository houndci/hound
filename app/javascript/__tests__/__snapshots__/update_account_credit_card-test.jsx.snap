exports[`test renders appropriately 1`] = `
<UpdateAccountCreditCard
  authenticity_token="csrf_token"
  stripe_customer_id_present={true} />
`;

exports[`test renders appropriately without Stripe customer ID 1`] = `
<UpdateAccountCreditCard
  authenticity_token="csrf_token"
  stripe_customer_id_present={false} />
`;
